package.json

```
@@ -13,22 +13,19 @@
   "license": "GPL-3.0-only",
   "author": "jellyfin-vue Contributors (https://github.com/jellyfin/jellyfin-vue/graphs/contributors)",
   "scripts": {
-    "analyze": "NUXT_SSR=1 nuxt build --analyze src",
-    "analyze:static": "nuxt build --analyze src",
-    "build": "NUXT_SSR=1 nuxt build src",
-    "build:standalone": "NUXT_SSR=1 nuxt build --standalone src",
-    "build:static": "nuxt build src",
-    "dev": "NUXT_SSR=1 nuxt src",
-    "dev:static": "nuxt src",
-    "generate": "nuxt generate src",
+    "analyze": "nuxt build --analyze src",
+    "analyze:ssr": "NUXT_SSR=1 nuxt build --analyze src",
+    "build": "nuxt build src",
+    "build:ssr": "NUXT_SSR=1 nuxt build src",
+    "build:ssr:standalone": "NUXT_SSR=1 nuxt build --standalone src",
+    "start": "nuxt src",
+    "start:ssr": "NUXT_SSR=1 nuxt src",
     "lint": "npm run lint:js && npm run lint:style",
     "lint:js": "eslint --ext .ts,.js,.json,.vue .",
     "lint:style": "stylelint **/*.{vue,css} --ignore-path .gitignore",
     "prepare": "husky install",
-    "prod": "npm run build src && npm run start src",
-    "prod:static": "npm run build:static src && npm run start:static src",
-    "start": "NUXT_SSR=1 nuxt start src",
-    "start:static": "nuxt start src",
+    "prod": "npm run build src && nuxt start src",
+    "prod:ssr": "npm run build:ssr src && NUXT_SSR=1 nuxt start src",
```

```suggestion
    "prod": "npm run build && nuxt start src",
    "prod:ssr": "npm run build:ssr && NUXT_SSR=1 nuxt start src",
```

I think it should be `npm run build` instead of `npm run build src` right?

---

No, because source code is under the "src" directory

---

But it's already in the built in npm command :)